### PR TITLE
Enhance ui to improve user experience

### LINKED ISSUE #89: Enhance Ui to improve user experience
Minor changes to Ui

### PR DESCRIPTION
**Pull request by Zhi-You**
Fixes #89

### CODE REVIEW

File: text-ui-test/EXPECTED.TXT [hunk 1]
```diff
@@ -28,15 +28,16 @@ The textbook for this module is available at:
 https://nus-cs2113-ay2021s1.github.io/website/se-book-adapted/index.html
 -------------------------------------------------------------------
 -------------------------------------------------------------------
-These are the available topics:
-OOP
+These are the available topics and the number of available questions in each:
+OOP   [1]
```
Comment (kstonekuan) [inline text-ui-test/EXPECTED.TXT:32]: Cool idea to add the number of questions

File: text-ui-test/EXPECTED.TXT [hunk 2]
```diff
@@ -69,5 +71,35 @@ This is the end of the quiz!
...
+1) A
+2) B
+3) C
+4) D
+-------------------------------------------------------------------
+-------------------------------------------------------------------
+Please choose the answer using its index (1,2,3 or 4)!
+-------------------------------------------------------------------
+-------------------------------------------------------------------
+Please choose only options 1, 2, 3 or 4
```
Comment (kstonekuan) [inline text-ui-test/EXPECTED.TXT:95]: Nice, good to check our exceptions

**Review by kstonekuan — APPROVED** [review]
LGTM

**Review by josephhhhhhhhh — APPROVED** [review]
Looks good overall, seems like the interactivity has been taken up which is really good! Solid enhancement.

File: src/main/java/seedu/eduke8/exception/ExceptionMessages.java [hunk 3]
```diff
@@ -5,16 +5,18 @@
...
     public static final String ERROR_UNKNOWN = "Oh no! An unknown error has occurred.";
     public static final String ERROR_UNRECOGNIZED_COMMAND = "Oops! The command cannot be understood."
-            + System.lineSeparator()
-            + "Please enter in \"help\" to get the list of available commands. ";
+            + System.lineSeparator() + "Please enter in \"help\" to get the list of available commands. ";
     public static final String ERROR_QUIZ_WRONG_FORMAT = "Invalid command! The command for quiz is as follows:"
-            + System.lineSeparator()
-            + "quiz t/<topic> n/<number of questions>";
+            + System.lineSeparator() + "quiz t/<topic> n/<number of questions>" + System.lineSeparator()
+            + "(Topics available can be found using the \"topics\" command)";
```
Comment (anqi20) [inline src/main/java/seedu/eduke8/exception/ExceptionMessages.java:11]: Oh thank you for adjusting the formatting!

File: src/main/java/seedu/eduke8/parser/QuizParser.java [hunk 4]
```diff
@@ -41,10 +44,10 @@ public Command parseCommand(DisplayableList optionList, String userInput) {
             return new AnswerCommand(chosenOption, question);
         } catch (NumberFormatException e) {
             LOGGER.log(Level.WARNING, "A non-number was given when answering question.");
-            return new IncorrectCommand("Please choose the answer by index");
+            return new IncorrectCommand(ERROR_QUIZ_ANSWER_NOT_INDEX);
         } catch (IndexOutOfBoundsException e) {
             LOGGER.log(Level.WARNING, "An index out of bounds was given when answering question.");
-            return new IncorrectCommand("Please choose only 1, 2, 3 or 4");
+            return new IncorrectCommand(ERROR_QUIZ_ANSWER_INDEX_OUT_OF_BOUNDS);
```
Comment (anqi20) [inline src/main/java/seedu/eduke8/parser/QuizParser.java:50]: Oh sorry I missed this out!

File: src/main/java/seedu/eduke8/ui/Ui.java [hunk 5]
```diff
@@ -39,7 +40,7 @@
             + System.lineSeparator() + "Hope you have learnt something new!";
     private static final String MESSAGE_ANSWER_WRONG = "Oops! The correct answer is ";
     private static final String MESSAGE_ANSWER_WRONG_SECOND = "! Do visit the textbook to read up more.";
-    private static final String MESSAGE_ANSWER_CORRECT = "Congrats! This answer is correct! Well Done!";
+    private static final String MESSAGE_ANSWER_CORRECT = "Great Job! That is the correct answer! Keep it up!";
```
Comment (anqi20) [inline src/main/java/seedu/eduke8/ui/Ui.java:43]: Oh, it sounds a lot nicer!

**Review by anqi20 — APPROVED** [review]

**Pull request merged**